### PR TITLE
feat(server): add datasetTag impl

### PR DESCRIPTION
**Pull request by fenngZ**
feat(server): add datasetTag impl

### CODE REVIEW

File: server/src/main/java/io/openbytes/DatasetTag.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+@Data
+@AllArgsConstructor
+public class DatasetTag {
+    /**
+     * tag name of this dataset
+     */
+    private String name;
+
+
```
Comment (ziyiyizi) [inline server/src/main/java/io/openbytes/DatasetTag.java:30]: 1

File: server/src/main/java/io/openbytes/DatasetTag.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openbytes;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
```
Comment (ziyiyizi) [inline server/src/main/java/io/openbytes/DatasetTag.java:22]: GETTER

File: server/src/main/java/io/openbytes/odi/application/TokenService.java [hunk 3]
```diff
@@ -48,4 +49,8 @@ public UserToken generateToken(String userId) {
         tokenRepository.save(oAuthToken);
         return oAuthToken;
     }
+
+    public User getUserByToken(String odiToken) {
+        return new User();
+    }
```
Comment (ziyiyizi) [inline server/src/main/java/io/openbytes/odi/application/TokenService.java:55]: todo next commit

File: server/src/main/java/io/openbytes/odi/domain/common/HttpURL.java [hunk 4]
```diff
@@ -40,9 +38,11 @@
     private final String url;
 
     public HttpURL(String url) {
-        if (!this.isUrl(url)) {
-            throw new BizException(CodeAndMessage.argsIllegal, "Not URL.");
+        if (StrUtil.isBlank(url)) {
+            this.url = "";
+            return;
         }
```
Comment (ziyiyizi) [inline server/src/main/java/io/openbytes/odi/domain/common/HttpURL.java:44]: ...

**Review by ziyiyizi — APPROVED** [review]

File: server/src/main/java/io/openbytes/DatasetTag.java (hunk 1 above)
Comment (fenngZ) [reply server/src/main/java/io/openbytes/DatasetTag.java:30]: done

File: server/src/main/java/io/openbytes/DatasetTag.java (hunk 2 above)
Comment (fenngZ) [reply server/src/main/java/io/openbytes/DatasetTag.java:22]: done

**Pull request merged**